src/main/java/com/plugish/woominecraft/WooMinecraft.java

```diff
@@ -47,25 +47,26 @@ public void onEnable() {
 			getLogger().warning( e.getMessage() );
 		}
 
-		this.lang = getConfig().getString( "lang" );
-		if ( lang == null ) {
+		setLang(getPluginConfig().getString( "lang" ));
+
+		if ( getLang() == null ) {
 			getLogger().warning( "No default l10n set, setting to english." );
-			this.lang = "en";
+			setLang("en");
 		}
 
 		initCommands();
-		getLogger().info( this.getLang( "log.com_init" ));
+		getLogger().info( this.getMessage( "log.com_init" ));
 
 		// Setup the scheduler
-		scheduler = new BukkitRunner( instance );
-		scheduler.runTaskTimerAsynchronously( instance, config.getInt( "update_interval" ) * 20, config.getInt( "update_interval" ) * 20 );
+		setScheduler(new BukkitRunner());

```

Comment:
Why set something and then use it immediately? I could understand if we were re-using the scheduler?